src/storage-client/src/healthcheck.rs

```diff
@@ -115,6 +115,10 @@ pub static MZ_STATEMENT_EXECUTION_HISTORY_DESC: Lazy<RelationDesc> = Lazy::new(|
         .with_column("cluster_id", ScalarType::String.nullable(true))
         .with_column("application_name", ScalarType::String.nullable(false))
         .with_column("cluster_name", ScalarType::String.nullable(true))
+        .with_column("transaction_isolation", ScalarType::String.nullable(false))
+        // Note that this can't be a timestamp, as it might be u64::max,
+        // which is out of range.
+        .with_column("execution_timestamp", ScalarType::UInt64.nullable(true))
```

Comment:
this will drop all the old data on the next release, so are there any other columns we really want to add?

Reply:
There are, but if I don't get to them by the next cut, it's not the end of the world (this feature is still in private preview)